test/Core.Test/AdminConsole/OrganizationFeatures/Policies/PolicyValidators/SingleOrgPolicyValidatorTests.cs

```diff
@@ -126,4 +127,69 @@ await sutProvider.GetDependency<IMailService>()
             .SendOrganizationUserRemovedForPolicySingleOrgEmailAsync(organization.DisplayName(),
                 "user3@example.com");
     }
+
+    [Theory, BitAutoData]
+    public async Task OnSaveSideEffectsAsync_GivenUpdatedSingleOrgPolicy_WhenAccountDeprovisioningIsDisabled_Then(
```

Comment:
I would either rename this to something clearer like `OnSaveSideEffectsAsync_WhenAccountDeprovisioningDisabled_DoesNotRevokeUsers ` or I would update the test above `OnSaveSideEffectsAsync_RemovesNonCompliantUsers` to assert that it does not revoke users